feign-form-spring/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    
+    <modelVersion>4.0.0</modelVersion>   
+    <packaging>jar</packaging>
+    
+    <artifactId>feign-form-spring</artifactId>
+    
+    <parent>
+        <groupId>io.github.openfeign.form</groupId>
+        <artifactId>feign-form-parent</artifactId>
+        <version>2.0.1</version>
+    </parent>
+    
+    <name>Open Feign Forms Extension for Spring</name>   
+    
+    <dependencies>
+        <dependency>
+            <groupId>ru.xxlabaza</groupId>
```

Comment:
1) ru.xxlabaza group id doesn't exists any more
2) you have always current project version